aptos-move/framework/table-natives/src/lib.rs

```diff
@@ -73,7 +73,7 @@ struct TableData {
 struct Table {
     handle: TableHandle,
     key_layout: MoveTypeLayout,
-    value_layout: MoveTypeLayout,
+    value_layout_info: (MoveTypeLayout, bool),
```

Comment:
if we add a bool like this let's have a small comment too.
we could also maybe add a member, but knowing it's legacy, I'm not sure what's the best, @igor-aptos any thoughts?

Reply:
agreed that the comment is minimal need.

Reply:
This is not just legacy, it is needed if we store something in a table which has a new aggregator as well.